src/components/Dropdown/DropdownInput.tsx

```diff
@@ -37,15 +37,19 @@ export function DropdownInput(props: {
 
   const handleChange = useCallback((e: ChangeEvent<HTMLInputElement>) => {
     toggleDropdown(true);
+    onChange?.(e.target.value);
```

Comment:
For my edification was there a reason we moved the onChange?

Reply:
We wanted to call the `onChange` earlier so it could fetch the new items in the dropdown before updating the focused item